fz_manager/main.py

```diff
@@ -181,6 +182,65 @@ def callback(monitor):
         except BaseException as ex:
             return await AlertMenu(str(ex) or ex.__class__.__name__).show()
 
+    async def upload_mods_menu_auto(self):
+        mods_folder_path = await PathMenu(
+            'Insert path to mods folder: ',
+            only_directories=True,
+            validator=lambda p: path.exists(p),
+            titlebar=self.titlebar,
+            load_last_value=True,
+            history=self.storage.mods_path_history
+        ).show()
+        if mods_folder_path is None:
+            return
+
+        root, _, filenames = next(walk(mods_folder_path), (None, None, []))
+        zip_files = list(filter(lambda n: n.endswith('.zip'), filenames))
+
+        if len(zip_files) == 0:
+            return await AlertMenu('No mod found in folder').show()
+
+        with open(f"{mods_folder_path}/mod-list.json", "r") as f:
+            mods = json.load(f)['mods']
+            mods.pop(0)
+
+        new_mods = []
+        for i in mods:
+            if i['enabled']:
+                new_mods.append(i)
+
+        zips, i=[], 0
+        for zip in zip_files:
+            zip = zip.strip('.zip')
+            if len(new_mods) == 0:
+                break
+            if zip == new_mods[i]['name']:
+                zips.append(zip)
+                if len(new_mods) == 1:
+                    break
+                i+=1
+
+        mods: list[FZClient.Mod] = []
+        for entry in zips:
```

Comment:
This section here duplicates code from `upload_mods_menu` and should be extracted to a shared method